App styling - Add style to the application

ram1117 opened the pull request:
Hi, in this PR, I have implemented the following in the project. Please review and provide feedback.

- Add styling to the applciation based on the design provided in the Zeplin template.
- Document project changes in readme file.

![image](https://user-images.githubusercontent.com/116718155/222162864-45b845a2-a7b6-42ef-8e91-abab57ebf1b8.png)

ram1117 committed:
Add styling to navigation bar

ram1117 committed:
add styling to each book component

ram1117 committed:
Finish final styling for form elements,  app component

ram1117 committed:
Update readme file

youmari commented on src/App.js:
```diff
@@ -9,7 +9,7 @@ const App = () => (
     <Navbar />
     <Routes>
       <Route index element={<Navigate to="/books" replace />} />
-      <Route path="/books" element={<Books />} />
+      <Route path="/books" element={<Books className="books-container" />} />
       <Route path="/categories" element={<Categories />} />
     </Routes>
   </div>
```
- [ ] Your Styling is as expected however you should remove the spaces between the pages  cause you should only take the page in the design not all on the image literally also it doesn't make sense for a website like that so please try to remove in order to complete the requirements
![Screenshot 2023-03-01 at 15 06 58](https://user-images.githubusercontent.com/61790587/222164591-b0fc5bfb-476c-44f0-bef2-7387e55470e7.png)

youmari reviewed (changes requested):
Hi @ram1117 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ram1117 committed:
remove background on body and set app width and height to 100%, based on review comments

ram1117 commented:
Implemented the changes requested by the previous reviewer. Removed the extra space outside the App component. 

![image](https://user-images.githubusercontent.com/116718155/222166562-7907d3d8-1508-4004-9c88-5b57a0a42955.png)

ShadyShawkat reviewed (approved):
## Hi @ram1117, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

![Image](https://media.giphy.com/media/jJQC2puVZpTMO4vUs0/giphy.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
